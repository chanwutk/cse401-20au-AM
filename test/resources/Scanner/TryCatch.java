
try {
  a + b;
} catch (RuntimeException exc) {
  a - b;
}

try {
  a + b;
} catch (Test exc) {
  a - b;
}