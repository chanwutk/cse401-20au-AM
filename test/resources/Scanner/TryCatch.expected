TRY LBRACKET ID(a) PLUS ID(b) SEMICOLON RBRACKET CATCH LPAREN RUNTIME_EXCEPTION ID(exc) RPAREN LBRACKET ID(a) MINUS ID(b) SEMICOLON RBRACKET TRY LBRACKET ID(a) PLUS ID(b) SEMICOLON RBRACKET CATCH LPAREN ID(Test) ID(exc) RPAREN LBRACKET ID(a) MINUS ID(b) SEMICOLON RBRACKET